go/pkg/tree/tree.go

```diff
@@ -378,11 +378,16 @@ func ComputeOutputsToUpload(execRoot string, paths []string, chunkSize int, cach
 		if err != nil {
 			return nil, nil, err
 		}
+		ch, err := chunker.NewFromProto(rootDir, chunkSize)
```

Comment:
> But for that you'd need to upload all the directory structure -- all intermediate Directory protos, including the root (something like packageTree does for input directories). Uploading only the rootPb will not be enough.

Yes. And without this change, chunkers returned from ComputeOutputsToUpload includes all intermidiate Directory/File protos except root directory.
But I'd like chunkers include root directory so that uploading chunkers returned from ComputeOutputsToUpload becomes sufficient to call GetTree.

Reply:
Sorry, missed this. Yes, you're right! I assume you verified that GetTree works with this change?

Reply:
> Sorry, missed this. Yes, you're right! I assume you verified that GetTree works with this change?

yes.